crates/sargon-uniffi/src/signing/transaction_intent/sign_request.rs

```diff
@@ -0,0 +1,81 @@
+use crate::prelude::*;
+use sargon::SignRequest as InternalSignRequest;
+use sargon::TransactionIntent as InternalTransactionIntent;
+
+#[derive(Clone, PartialEq, Eq, uniffi::Record)]
+pub struct SignRequestForTransactionIntent {
+    pub factor_source_kind: FactorSourceKind,
+
+    /// Per factor source, a set of transactions to sign, with
+    /// multiple derivations paths.
+    pub per_factor_source: HashMap<
```

Comment:
no HashMap, removes our needed order. see https://github.com/radixdlt/sargon/pull/288/files#r1866012477

use macro to declare this and the other :)